.env

```diff
@@ -52,3 +52,4 @@ LDAP_PEOPLE_OU='ou=users,dc=domain,dc=org'
 ACCOUNT_LESS_STRICT_PASSWORD_RULES=FALSE
 MAILER_FROM_ADDR='email@domain.null'
 MAILER_FROM_NAME='SenderName'
+PELAGOS_NAME=pelagos
```

Comment:
This is just a weird parameter. Is this different in env.local?

Reply:
The value is the pelagos name of our instance, which you need to add in the .env.local file
For example:
PELAGOS_NAME=pelagos.ppondicherry